README.md

```diff
@@ -198,6 +198,25 @@ You can also use `Authorization: API-KEY 7b3b910b-a7ad-41e8-89d6-5e28e2e34e70` h
 
 You create a private box by pushing your first record to a new box with an API-KEY. All the subsequent write requests to that box expect the API-KEY to be passed. _You cannot change a public box to private or vice versa._
 
+### Getting Box metadata
+
+Use `/_meta/BOX_ID` in a `GET` request to get metadata of a box
+```
+https://jsonbox.io/_meta/demobox_6d9e326c183fde7b
+```
+
+The result will have the following format
+```json
+{
+    "count": 3,
```

Comment:
Can we add a prefix `_` to all the fields to be consistent across all the routes? Also, `_` means its a system fields, so we can easily distinguish it from the user data.